plugin/cache.go

```diff
@@ -222,14 +222,12 @@ func cachedRead(ctx context.Context, e Entry) (entryContent, error) {
 			}
 			return newEntryContent(rawContent), nil
 		case blockReadableSignature:
-			// Go doesn't allow overloaded functions, so the external plugin entry type
```

Comment:
You decided not to keep this explanation?

Reply:
I moved it over to where `blockRead`'s declared

Reply:
Oh, just moved it.